TYPO3.Neos/Classes/TYPO3/Neos/Eel/FlowQueryOperations/SortOperation.php

```diff
@@ -0,0 +1,117 @@
+<?php
+namespace TYPO3\Neos\Eel\FlowQueryOperations;
+
+/*
+ * This file is part of the TYPO3.Neos package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Eel\FlowQuery\FlowQuery;
+use TYPO3\Eel\FlowQuery\Operations\AbstractOperation;
+use TYPO3\Flow\Annotations as Flow;
+use TYPO3\TYPO3CR\Domain\Model\Node;
+use TYPO3\TYPO3CR\Domain\Model\NodeInterface;
+
+/**
+ * "sort" operation working on TYPO3CR nodes.
+ * Sorts nodes by specified node properties.
+ */
+class SortOperation extends AbstractOperation
+{
+
+    /**
+     * {@inheritdoc}
+     *
+     * @var string
+     */
+    protected static $shortName = 'sort';
+
+    /**
+     * {@inheritdoc}
+     *
+     * @var integer
+     */
+    protected static $priority = 1;
+
+    /**
+     * {@inheritdoc}
+     *
+     * We can only handle TYPO3CR Nodes.
+     *
+     * @param mixed $context
+     * @return boolean
+     */
+    public function canEvaluate($context)
+    {
+        return count($context) === 0 || (isset($context[0]) && ($context[0] instanceof NodeInterface));
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * First argument is the node property to sort by. Works with internal arguments (_xyz) as well.
+     * Second argument is the sort direction (ASC or DESC).
+     *
+     * @param FlowQuery $flowQuery the FlowQuery object
+     * @param array $arguments the arguments for this operation.
+     * @return mixed
+     */
+    public function evaluate(FlowQuery $flowQuery, array $arguments)
+    {
+        $nodes = $flowQuery->getContext();
+
+        // Check sort property
+        if (isset($arguments[0]) && !empty($arguments[0])) {
+            $sortProperty = $arguments[0];
+        } else {
+            throw new \TYPO3\Eel\FlowQuery\FlowQueryException('Please provide a node property to sort by.', 1467881104);
+        }
+
+        // Check sort direction
+        if (isset($arguments[1]) && !empty($arguments[1]) && in_array($arguments[1], array('ASC', 'DESC'))) {
+            $sortOrder = $arguments[1];
```

Comment:
I think it should be a bit more forgiving and accept stuff like `asc` as well.. just strtolower it..